Makefile

```diff
@@ -1,11 +1,17 @@
 SHELL := /bin/bash
 
+
```

Comment:
@cgruver [nit] This extra line is not needed.

Reply:
That's true.  Superfluous white space.

Reply:
Unless, you want me to add a commit to remove that extra line, I'll save it for a follow-up PR for the Makefile based on an idea floated by Guillaume.